src/cpp/src/image_generation/stable_diffusion_3_pipeline.hpp

```diff
@@ -456,7 +452,13 @@ class StableDiffusion3Pipeline : public DiffusionPipeline {
     }
 
     void set_lora_adapters(std::optional<AdapterConfig> adapters) override {
-        OPENVINO_THROW("LORA adapters are not implemented for Stable Diffusion 3 yet");
+        if(adapters) {
+            if(auto updated_adapters = derived_adapters(*adapters)) {
+                adapters = updated_adapters;
+            }
+            //m_clip_text_encoder->set_adapters(adapters);
```

Comment:
Please add suport for LoRAs with text encoder(s), or remove this commented line, or add `TODO` comment

Reply:
Add TODO comment for SD3 LoRA support for text encoder(s)